test/adapter/json_api/linked_test.rb

```diff
@@ -1,11 +1,16 @@
 require 'test_helper'
+
+NestedPost = Class.new(Model)
+class NestedPostSerializer < ActiveModel::Serializer
```

Comment:
Yes. Let's leave them there for now, and take care of it during the Great Test Refactor.